command-page.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <!--Headers for both the page itself and the navbar-->
+    <h1 class="title"> Command Line </h1>
+    <h1 class="nav-bar"></h1>
+
+    <!--Main contains both the info article and the aside for syntax-->
+    <main class="terminal">
+
+        <!--Article that contains two sections, one for a description and one for links-->
+        <article class="info">
+            <section class="info__description">
+                <h3 class="info__description-header"> What is a terminal? </h3>
+                <p class="info__description-text">Terminal, sometimes referred to as <em>console</em> or
+                    <em>command line</em>, is a built-in tool that allows you to type instructions in order
+                    to navigate and modify content on your computer. With it, you can explore, change, and delete
+                    directories, files, and other apps and programs. The terminal not only
+                    accesses your local computer, but can also connect and sync to online accounts. For web developers,
+                    the most common external program that is used in tandem with terminal is gitHub. See links below for
+                    more information on what gitHub is and how to use it.
+                </p>
+
+            </section>
+            <!--add a header?-->
+            <section class="links">
+                <h3 class="links-header">Further Resources</h3>
+                <p class="link-urls">
+                <div>1. Watch <a href="https://youtu.be/5XgBd6rjuDQ">this video</a> to learn more about how to use
+                    command line. </div>
+                <div>2. A brief introduction to <a href="https://youtu.be/uR6G2v_WsRA">git core concepts.</a>
+                </div>
+                <div>3. What in the world is <a href="https://guides.github.com/activities/hello-world/">gitHub??</a></div>
+                </p>
+
+
+            </section>
+        </article class="glossary">
+        <!--The aside fot the syntax bar-->
+        <aside class="glossary__terms">
+            <h3>Useful terms and tips:</h3>
```

Comment:
Might want to add a class for the header named "syntax-box__header" but not sure if that's required or not yet.